app/components/finance/npq/payment_overviews/payment_overview_table_header.html.erb

```diff
@@ -2,6 +2,6 @@
   <% milestones.order(name: :asc).each do |milestone| %>
     <%= render Finance::NPQ::PaymentOverviews::PaymentOverviewMilestoneSummary.new(milestone, total_participants_for(milestone)) %>
   <% end %>
-  <span class="govuk-body-s"><%= t("finance.total_declarations") %><br><%= total_declarations %></span>
+  <span class="govuk-body-s"><%= t("finance.total_declarations") %><br><%= total_declarations(npq_lead_provider, contract) %></span>
```

Comment:
from a product point of view if a statement had ineligibles attached should they be counted towards `total_declarations?`

Reply:
Just checking with Rhys

Reply:
They shouldn't be included, it should only be the 'payable' ones